jstreegrid.js

```diff
@@ -41,11 +41,13 @@
     getIds = function (nodes) {
       return $.makeArray(nodes.map(function(){return this.id;}));
     },
-		findDataCell = function (uniq, ids, col) {
-			var ret = $(), columns = [].concat(col), cellId;
+    findDataCell = function (uniq, ids, col, scope) {
```

Comment:
You need the scope, because we want wo search outside the document wihtin the detached columns! Just using $() - as it was before - would not find the nodes in the detached columns.
So when we search outside the document, we have to provide a scope to search in!